test/Microsoft.Azure.ServiceBus.MessageId.Test/When_MessageIdPlugin_is_used.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
This line looks right. There's a copyright line which got missed while copying i suppose. Add the below line in the beginning of the file.
`// Copyright (c) Microsoft. All rights reserved.`